### ISSUE TITLE
I can no longer integrate the Analytics service.

### ISSUE THREAD

**Issue by rronchi-dev** [google/site-kit-wp#705]
## Bug Description
After upgrading to version to version 1.0.0-rc.1.1, I can no longer integrate the Analytics service.

## Steps to reproduce
I reinstalled the Plugin and after trying to setup the Analytics, I had access to the Analytics page in the menu, but it appears empty (without any content).
In Settings appears "Analytics is not connected" and the option "Setup incomplete: continue module setup", clicking on the option it displays the same page without content.

## Screenshots
In the Console of WebTools the following 3 errors appear:
1.)
navigation.js:28 Error: Loading chunk 6 failed.
(missing: https://starchi.com.br/wp-content/plugins/google-site-kit/dist/assets/js/chunk-googlesitekit-setup-wrapper.js)
    at HTMLScriptElement.i (googlesitekit-module.js?ver=1.0.0-rc.1.1:1)
t.<computed> @ navigation.js:28

2.)
Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Error%3A%20Loading%20chunk%206%20failed.%0A(missing%3A%20https%3A%2F%2Fstarchi.com.br%2Fwp-content%2Fplugins%2Fgoogle-site-kit%2Fdist%2Fassets%2Fjs%2Fchunk-googlesitekit-setup-wrapper.js)&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at pb (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:12:454)
    at n (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:13:221)
    at Ac (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:70:369)
    at https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:80:190
    at N (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:90:3)
    at hi (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:105:408)
    at Qg (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)
    at Z (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:156:492)
    at Kc (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:155:52)
    at li (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:152:175)
t.<computed> @ navigation.js:28

3.)
Uncaught (in promise) Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Error%3A%20Loading%20chunk%206%20failed.%0A(missing%3A%20https%3A%2F%2Fstarchi.com.br%2Fwp-content%2Fplugins%2Fgoogle-site-kit%2Fdist%2Fassets%2Fjs%2Fchunk-googlesitekit-setup-wrapper.js)&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at pb (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:12:454)
    at n (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:13:221)
    at Ac (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:70:369)
    at https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:80:190
    at N (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:90:3)
    at hi (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:105:408)
    at Qg (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)
    at Z (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:156:492)
    at Kc (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:155:52)
    at li (https://starchi.com.br/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:152:175)

## Additional Context
- WordPress Version: 5.2.4
- PHP Version: 7.1
- OS: in the server, I think is LINUX
- Browser: chrome
- Plugin Version:  1.0.0-rc.1.1
- Device: Desktop/Windows 10
## Acceptance criteria

* 

## Implementation Brief

* 

## Changelog entry

*

**Comment by ernee** [comment]
@rronchi-dev thanks for the details. Could you try these troubleshooting steps:

1. Reset Site Kit (Site Kit > Settings > Admin Settings > Reset Site Kit)
2. Temporarily deactivate any caching or security suite plugins in use
3. Install Site Kit in a new [incognito window](https://i.imgur.com/ALzgiHp.png)  

If the above fails please share your [Site Health info](https://i.imgur.com/Rtkmsba.png)

**Comment by ernee** [comment]
@rronchi-dev just a quick follow up that I was able to recreate the issue and resolve it in my case.

It may be related to browser caching, so along with the above troubleshooting steps also try clearing or disabling your browser cache. The errors should resolve and the page should render.

Let me know how it goes!

**Comment by rronchi-dev** [comment]
Hi,

By following the steps below, I was able to reconfigure the site-kit and integrate with all the services I use.

Many thanks for the guidelines!

1. Reset Site Kit (Site Kit > Settings > Admin Settings > Reset Site Kit)
2. Temporarily deactivate any caching or security suite plugins in use
3. Install Site Kit in a new incognito window


De: Renee Johnson <notifications@github.com>
Enviada em: quinta-feira, 24 de outubro de 2019 16:13
Para: google/site-kit-wp <site-kit-wp@noreply.github.com>
Cc: R Ronchi <rronchi@starchi.com.br>; Mention <mention@noreply.github.com>
Assunto: Re: [google/site-kit-wp] I can no longer integrate the Analytics service. (#705)


@rronchi-dev<https://github.com/rronchi-dev> just a quick follow up that I was able to recreate the issue and resolve it in my case.

It may be related to browser caching, so along with the above troubleshooting steps also try clearing or disabling your browser cache. The errors should resolve and the page should render.

Let me know how it goes!
